src/main/java/com/google/sps/servlets/AggregationServlet.java

```diff
@@ -24,8 +34,54 @@
 @WebServlet("/aggregate")
 public class AggregationServlet extends HttpServlet {
 
+  private static final Set<String> aggregableFields =
+      new HashSet<>(Arrays.asList("annotatedAssetId, annotatedLocation, annotatedUser"));
```

Comment:
Perhaps simpler to just use Set.of(...)

Reply:
I got rid of the aggregableFields set entirely because I realized it is unnecessary.  The AnnotatedField.create() method already knows what the valid field names are and can just throw an error when it receives an invalid name.